draft-ietf-webtrans-http2.md

```diff
@@ -253,9 +253,21 @@ single HTTP/2 connection.  The client MUST NOT open more sessions than
 indicated in the server SETTINGS parameters.  The server MUST NOT close the
 connection if the client opens sessions exceeding this limit, as the client and
 the server do not have a consistent view of how many sessions are open due to
-the asynchronous nature of the protocol; instead, it MUST reply to the CONNECT
-request with a status code 429, indicating that the client attempted to open
-too many sessions.
+the asynchronous nature of the protocol; instead, it MUST reset all of the
+CONNECT streams it is not willing to process with the `REFUSED_STREAM`
+error code ({{Section 8.7 of HTTP2}}).
```

Comment:
I'm no longer sure about this entire paragraph...

A server will only be unsure if it reduces the limit and the SETTINGS frame hasn't been acknowledged yet.  But then it is sure, because everything opened before the SETTINGS+ACK uses the larger limit and everything after uses the reduced limit...  So maybe we can completely rewrite this little bit.

Reply:
Yeah, I think in H2 it's much more clear cut and deterministic. Let's spin out an issue -- we've aligned the error response when we need to say "no, stop", but we may not want to exactly match where the underlying protocol _really_ doesn't match.

Filed #94